organizerguide.md

```diff
@@ -2,6 +2,9 @@
 
 So, you are thinking of organizing a short virtual conference session. This is easy, rewarding, and takes little work. This manual will guide you through the process.
 
+## Main points to consider
+---
```

Comment:
The line separators are meant to separate sections of a document. Are you sure one is appropriate here?

Reply:
I felt that the headers of the different section levels were not distinguishable enough, so the lines are a suggestion to help make those demarcations more clear.

Reply:
We should rather change the style his css at a later stage then